app/components/todo-item.hbs

```diff
@@ -0,0 +1,24 @@
+<li>
+  <input
+    class="todoTitleInput"
+    aria-label={{@todo.title}}
+    {{on "blur" (fn @onEditTaskTitle @todo.id)}}
+    value={{@todo.title}}
+  />
+  {{#each @statusActions as |action|}}
+    <button
+      class="{{action.buttonClass}}"
+      type="button"
+      {{on "click" (fn @updateStatus @todo.id action.status)}}
```

Comment:
Why do we need an `on` modifier ?

Reply:
added on modifier to get click event from the button. Changing the status of the task using this on click event

Reply:
Please go through the inbuilt `action` helper and use it. Lets not add new packages and make the app heavy.

Reply:
As we discussed, the "action" helper is not supported in 3.2.7. Also, "on" is inbuilt modifier.